### DISCUSSION TITLE
Session expiration in Serenity and handling http 400 error

### DISCUSSION THREAD

**Discussion by zbouchlih** [serenity-is/Serenity#6235] [Help]
Hi All,

I am using an active directory authentication with serenity and after 1h of opening the page of list, if I try to click on an Item I receive this error : 

![image](https://user-images.githubusercontent.com/9480278/150825395-8e1d2b85-6b23-460c-b073-8eb1b808c86f.png)

The error in the console is :  

Failed to load resource: the server responded with a status of 400 (Bad Request)
:10031/services/Service/Retrieve:1 

To fix that I need to refresh the page

in my Startup.cs, I have this code :  

services.AddAuthentication(o =>
            {
                o.DefaultScheme = CookieAuthenticationDefaults.AuthenticationScheme;
                o.DefaultAuthenticateScheme = CookieAuthenticationDefaults.AuthenticationScheme;
                o.DefaultChallengeScheme = CookieAuthenticationDefaults.AuthenticationScheme;
            }).AddCookie(o =>
            {
                o.Cookie.Name = ".AspNetAuth";
                o.LoginPath = new PathString("/Account/Login/");
                o.AccessDeniedPath = new PathString("/Account/AccessDenied");
                o.ExpireTimeSpan = TimeSpan.FromHours(7);
                o.SlidingExpiration = true;
            });

But chagning the ExpireTimeSpan seems not affecting anything, I guess because Active directory authentication

- Do you have any idea how I can remove that time limit please ?
- Can I at least change the error descripton ? How can I handle that to change the text ??

This error appear only on the version deployed on IIS, I don't receive an error in localhost

Thanks a lot for your help

**Comment by osdan-madrid** [comment]
Did you find a solution? I've the same problem.
